specifications/terminology.md

```diff
@@ -1,3 +1,17 @@
 # Terminology
 
```

Comment:
Currently we define our terminology over multiple documents, e.g. also in the CIP and VPP specifications. What about removing the Terms section in all other documents and just referring to this markdown file?